src/spyglass/common/common_filter.py

```diff
@@ -167,9 +167,9 @@ def add_filter(
     def _filter_restrict(self, filter_name, fs):
         return (
             self & {"filter_name": filter_name} & {"filter_sampling_rate": fs}
-        ).fetch1(as_dict=True)
```

Comment:
is this just because `as_dict=True` is redundant?

Reply:
Yes, `fetch1` always returns as dict